pkgs/applications/science/misc/root/default.nix

```diff
@@ -151,40 +151,18 @@ stdenv.mkDerivation rec {
   '';
 
   cmakeFlags = [
-    "-Drpath=ON"
```

Comment:
This is probably okay to remove, if it's now default ON on all platforms.

Reply:
See https://github.com/NixOS/nixpkgs/pull/308497#discussion_r1588474977